Orange/preprocess/transformation.py

```diff
@@ -107,7 +107,10 @@ def __init__(self, variable, offset, factor):
         self.factor = factor
 
     def transform(self, c):
-        return (c - self.offset) * self.factor
+        if sp.issparse(c):
+            return c
```

Comment:
I'd then suggest that we normalize as I suggested above; the method must not silently fail, hence is must use the factor (because it can) and raise an error when offset is non-zero (because it can't do anything else).

The problem with the SVM is not directly related and can be solved in another PR.